libraries/SITL/SITL.cpp

```diff
@@ -65,8 +65,14 @@ SIM *SIM::_singleton = nullptr;
 
 // table of user settable parameters
 const AP_Param::GroupInfo SIM::var_info[] = {
-    
+
+    // @Param: DRIFT_SPEED
+    // @DisplayName: Gyro drift speed
+    // @Description: Gyro drift rate of change in degrees/second/minute
```

Comment:
Are these *really* in minutes?!  That's rather odd.  Not saying you're wrong, but please make sure its right!

Also, we can put units into separate fields, like this:

```suggestion
    // @Description: Gyro drift rate of change
    // @Units: deg/s/s
```

Known units are here: https://github.com/ardupilot/ardupilot/blob/master/Tools/autotest/param_metadata/param.py#L66

Reply:
It is indeed uncommon. I referred to the `SITL.h` where the comment states:     `AP_Float drift_speed; // degrees/second/minute`
Also, I see `    double minutes = fmod(AP_HAL::micros64() / 60.0e6, period);
    if (minutes < period/2) {
        return minutes * ToRad(sitl->drift_speed);
    }
    return (period - minutes) * ToRad(sitl->drift_speed);`

Reply:
Great, thanks!

Sometimes the hard-to-believe things just make you do more work :-)